test/api_test.py

```diff
@@ -1,14 +1,16 @@
 import unittest
+import configparser
 from glusterapi import Client
 
 class TestPeer(unittest.TestCase):
 	@classmethod
-	def setUpClass(self):
-		self.client = Client("10.70.42.00")
+	def setUpClass(cls):
+		cls.config = configparser.ConfigParser()
+		cls.config.read('config.toml')
```

Comment:
ConfigParser isn't really a TOML parser. In order to avoid confusion I think its proabably easy enough to rename your config file to ini (or anything that doesn't mislead future people that the file supports toml).

Reply:
@phlogistonjohn right, my bad. will change that. Thank you.

Reply:
@devyanikota you should be able to read all configurable parameters from the config file.
something like this.

config.ini
```
[glusterd2]
endpoint: http://127.0.0.1:24007
user: glustercli
secret: dc12b44485e806975985853b1af0a23165edc799023865801665f25bfe03e1e9
verify: False

[peer]
host1: 10.60.43.235:24008
host2: 10.60.42.235:24008

[devices]

```

```
endpoint = config.get('glusterd2', 'endpoint')
user = config.get('glusterd2', 'user')
secret = config.get('glusterd2', 'secret')
verify = config.getboolean('glusterd2', 'verify')
host = config.get('peer', 'host')
```